### DISCUSSION TITLE
How to use existingSecret

### DISCUSSION THREAD

**Discussion by dkr91** [renovatebot/renovate#26566] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

CE 6.8.0

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### What is your question?

I have an old Renovate 5.1.0 instance running in OpenShift and want to migrate to CE 6.8.0. Can I parse my existing Secrets I configured in OpenShift through changing their name and then filling the name of the Secrets in the existingSecret field in the values.yaml or do I have to specify each field of my pre-existing secrets?

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by rarkins** [comment]
I think this is meant to be in https://github.com/mend/renovate-ce-ee and not this repo. Unfortunately GitHub doesn't allow transferring discussions between separate orgs, so I can only close this.
